lib/src/main/java/org/team100/lib/commands/drivetrain/DriveToPoseWithProfile.java

```diff
@@ -22,18 +22,18 @@
  */
 public class DriveToPoseWithProfile extends Command implements Glassy {
     private final FieldLogger.Log m_field_log;
-    private final Supplier<Optional<Pose2d>> m_goals;
+    private final Supplier<Optional<SwerveModel>> m_goals;
     private final SwerveDriveSubsystem m_drive;
     private final SwerveController m_controller;
     private final HolonomicProfile m_profile;
 
-    private Pose2d m_goal;
+    private SwerveModel m_goal;
     private ProfileReference m_reference;
     private ReferenceController m_referenceController;
 
     public DriveToPoseWithProfile(
             FieldLogger.Log fieldLogger,
-            Supplier<Optional<Pose2d>> goals,
+            Supplier<Optional<SwerveModel>> goals,
```

Comment:
why isn't this just a single goal?  when would you change the goal after creating this object?  why is it optional?